README.rst

```diff
@@ -61,46 +78,49 @@ __ https://docs.scale.com/reference#general-image-annotation
         }
     )
 
-Retrieve task
-^^^^^^^^^^^^^
+Retrieve a task
+^^^^^^^^^^^^^^^
 
 Retrieve a task given its id. Check out `Scale's API documentation`__ for more information.
 
 __ https://docs.scale.com/reference#retrieve-tasks
 
 .. code-block :: python
 
-    task = client.fetch_task('asdfasdfasdfasdfasdfasdf')
-    print(task.status)  // Task status ('pending', 'completed', 'error', 'canceled')
-    print(task.response) // If task is complete
+    task = client.get_task('30553edd0b6a93f8f05f0fee')
+    print(task.status)  # Task status ('pending', 'completed', 'error', 'canceled')
+    print(task.response) # If task is complete
 
 List Tasks
 ^^^^^^^^^^
 
-Retrieve a list of tasks, with optional filter by start and end date/time. Paginated with `next_token`. The return value is a `scaleapi.Tasklist`, which acts as a list, but also has fields for the total number of tasks, the limit and offset, and whether or not there's more. Check out `Scale's API documentation`__ for more information.
+Retrieve a list of `Task` objects, with optional filters for: `project_name, batch_name, type, status, review_status, unique_id, completed_after, completed_before, updated_after, updated_before, created_after, created_before` and `tags`.
+
+This method is a generator and yields tasks. It can be wrapped in a `list` statement if a Task list is needed.
```

Comment:
Can we add a few more sentences here about how a generator works / the implications. A lot of less experienced developers will also be using this method so we'll want to give at least a little bit of background about how to take advantage of this.